### PR TITLE
add R support

### PR DESCRIPTION
**Pull request by gengjiawen**
Fixes https://github.com/tabrindle/envinfo/issues/96.

### CODE REVIEW

File: src/helpers/languages.js [hunk 1]
```diff
@@ -58,6 +58,14 @@ module.exports = {
     ]).then(v => utils.determineFound('Python', v[0], v[1]));
   },
 
+  getRInfo: () => {
+    utils.log('trace', 'getRInfo');
+    return Promise.all([
```
Comment (tabrindle) [inline src/helpers/languages.js]: wow does this really work on windows linux and macos?

Comment (gengjiawen) [reply src/helpers/languages.js]: I test it on linux.

Comment (gengjiawen) [reply src/helpers/languages.js]: See https://circleci.com/gh/gengjiawen/envinfo/214?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link.

Comment (gengjiawen) [reply src/helpers/languages.js]:
```console
  System:
    OS: Linux 4.15 Ubuntu 19.04 (Disco Dingo)
    CPU: (36) x64 Intel(R) Xeon(R) Platinum 8124M CPU @ 3.00GHz
    Memory: 944.74 MB / 68.69 GB
    Container: Yes
    Shell: 5.0.3 - /bin/bash
  Binaries:
    Node: 12.3.1 - /usr/local/bin/node
    Yarn: 1.16.0 - /usr/local/bin/yarn
    npm: 6.9.0 - /usr/local/bin/npm
  Utilities:
    CMake: 3.10.2 - /usr/local/bin/cmake
    Make: 4.2.1 - /usr/bin/make
    GCC: 8.3.0 - /usr/bin/gcc
    Git: 2.20.1 - /usr/bin/git
  IDEs:
    Vim: 8.1 - /usr/bin/vim
  Languages:
    Bash: 5.0.3 - /usr/bin/bash
    Perl: 5.28.1 - /usr/bin/perl
    Python: 2.7.16 - /usr/bin/python
    Python3: 3.7.3 - /usr/bin/python3
    R: 3.5.2
```

Comment (tabrindle) [reply src/helpers/languages.js]: is there no path available?

Also I don't doubt you tested on linux, but my point is if this hasnt been tested on macOS or windows, we might want an OS conditional around this until it is.

Comment (gengjiawen) [reply src/helpers/languages.js]: > is there no path available?
Fixed.
> Also I don't doubt you tested on linux, but my point is if this hasnt been tested on macOS or windows, we might want an OS conditional around this until it is.

I install R on appveyor, let's see it on CI.
As for macOS, the output is similar on linux. I will test the command on machine.

Comment (gengjiawen) [reply src/helpers/languages.js]: Looks not working on windows, I will see what I can do.

Comment (tabrindle) [reply src/helpers/languages.js]: Tested on macos

```
➜  ~ R --version
R version 3.6.0 (2019-04-26) -- "Planting of a Tree"
Copyright (C) 2019 The R Foundation for Statistical Computing
Platform: x86_64-apple-darwin18.5.0 (64-bit)

R is free software and comes with ABSOLUTELY NO WARRANTY.
You are welcome to redistribute it under the terms of the
GNU General Public License versions 2 or 3.
For more information about these matters see
https://www.gnu.org/licenses/.

➜  ~ which R
/usr/local/bin/R
➜  ~ 
```

I think we're good for that.

Comment (gengjiawen) [reply src/helpers/languages.js]: Works on windows too. See https://ci.appveyor.com/project/tabrindle/envinfo/builds/25037005.

It's like java, they write the version to error stream, took me a while to find the bug ...

Comment (gengjiawen) [reply src/helpers/languages.js]: Verify on macOS too.

**Comment by tabrindle** [comment]
Thanks for this 🎉

**Pull request merged**